src/exercises/2-Design_Tokens/Readme.md

```diff
@@ -39,9 +41,9 @@ import theme from './theme';
 <Typography theme={theme} />;
 ```
 
-## 2.2. Using design tokens in components
+## 2.2. Accessing design tokens from components
 
-styled-components makes it convenient to use design tokens, thanks to [ThemeProvider](https://github.com/component-driven/component-driven-development/blob/master/src/ThemeProvider.js) and React Context. You can access them in your component like this:
+styled-components makes it simple to use design tokens, thanks to [ThemeProvider](https://github.com/component-driven/component-driven-development/blob/master/src/ThemeProvider.js) and React Context. You can access them in your component like this:
```

Comment:
😢

Reply:
? You feel attached to previous version? I can change it back.

Reply:
I'm not attached to it, but telling people what should be simple for them isn't great. For some it still may not be simple, even if it's simpler than some other methods.

Reply:
Ah I see what you're saying. I agree to that. Going to change back.